### ISSUE TITLE
Error creating client with confluent cloud and 0.2.7.

### ISSUE THREAD

**Issue by rojopolis** [Mongey/terraform-provider-kafka#137]
Technical details below...
The below config with provider versions 0.2.5 & 0.2.6, but fails with 0.2.7.  I also tried setting `skip_tls_verify = true` but the error was the same.

I can see that the failing function [parsePemOrLoadFromFile](`https://github.com/Mongey/terraform-provider-kafka/compare/v0.2.6...v0.2.7#diff-2d9f0faf2880ac6dcd1ee6372c45a05aR83`) is new in this version, but I don't understand the underlying changes that require it.

Is this a config error on my part, or was there a breaking change in the provider?  If the latter, can you suggest a config that might work with Confluent Cloud?

Terraform version:
```
Terraform v0.12.24
+ provider.kafka v0.2.7
```

```HCL
provider "kafka" {
  version           = "~> 0.2"
  bootstrap_servers = ["pkc-4kgmg.us-west-2.aws.confluent.cloud:9092"]

  tls_enabled    = true
  sasl_username  = var.sasl_user_name
  sasl_password  = var.sasl_password
  sasl_mechanism = "plain"
  skip_tls_verify = true
}

resource "kafka_topic" "test" {
  name               = "test"
  replication_factor = 3
  partitions         = 2

  config = {
    "segment.ms"     = "86500000"
    "cleanup.policy" = "delete"
  }
}
```

terraform plan:
```
------------------------------------------------------------------------

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # kafka_topic.test will be created
  + resource "kafka_topic" "test" {
      + config             = {
          + "cleanup.policy" = "delete"
          + "segment.ms"     = "86500000"
        }
      + id                 = (known after apply)
      + name               = "test"
      + partitions         = 2
      + replication_factor = 3
    }

Plan: 1 to add, 0 to change, 0 to destroy.
```
terraform apply:
```
kafka_topic.test: Creating...

Error: open : no such file or directory

  on pksd.tf line 12, in resource "kafka_topic" "test":
  12: resource "kafka_topic" "test" {
```
debug output:
```
2020/06/29 11:44:50 [INFO] backend/local: apply calling Apply
2020/06/29 11:44:50 [INFO] terraform: building graph: GraphTypeApply
2020/06/29 11:44:50 [DEBUG] Resource state not found for node "kafka_topic.test", instance kafka_topic.test
2020/06/29 11:44:50 [DEBUG] ProviderTransformer: "kafka_topic.test (prepare state)" (*terraform.NodeApplyableResource) needs provider.kafka
2020/06/29 11:44:50 [DEBUG] ProviderTransformer: "kafka_topic.test" (*terraform.NodeApplyableResourceInstance) needs provider.kafka
2020/06/29 11:44:50 [DEBUG] ReferenceTransformer: "kafka_topic.test" references: []
2020/06/29 11:44:50 [DEBUG] ReferenceTransformer: "provider.kafka" references: []
2020/06/29 11:44:50 [DEBUG] ReferenceTransformer: "kafka_topic.test (prepare state)" references: []
2020/06/29 11:44:50 [DEBUG] Starting graph walk: walkApply
2020-06-29T11:44:50.941-0700 [INFO]  plugin: configuring client automatic mTLS
2020-06-29T11:44:50.969-0700 [DEBUG] plugin: starting plugin: path=/Users/rojo/Workspace/parkside-securities/terraform-provider-kafka/examples/pksd/terraform-provider-kafka_v0.2.7 args=[/Users/rojo/Workspace/parkside-securities/terraform-provider-kafka/examples/pksd/terraform-provider-kafka_v0.2.7]
2020-06-29T11:44:50.972-0700 [DEBUG] plugin: plugin started: path=/Users/rojo/Workspace/parkside-securities/terraform-provider-kafka/examples/pksd/terraform-provider-kafka_v0.2.7 pid=21360
2020-06-29T11:44:50.972-0700 [DEBUG] plugin: waiting for RPC address: path=/Users/rojo/Workspace/parkside-securities/terraform-provider-kafka/examples/pksd/terraform-provider-kafka_v0.2.7
2020-06-29T11:44:50.983-0700 [INFO]  plugin.terraform-provider-kafka_v0.2.7: configuring server automatic mTLS: timestamp=2020-06-29T11:44:50.983-0700
2020-06-29T11:44:51.012-0700 [DEBUG] plugin: using plugin: version=5
2020-06-29T11:44:51.012-0700 [DEBUG] plugin.terraform-provider-kafka_v0.2.7: plugin address: address=/var/folders/yj/_rppsd1x3hgcvt5j621szntr0000gn/T/plugin348764183 network=unix timestamp=2020-06-29T11:44:51.012-0700
2020-06-29T11:44:51.076-0700 [DEBUG] plugin.terraform-provider-kafka_v0.2.7: 2020/06/29 11:44:51 [DEBUG] configuring provider with Brokers @ &[pkc-4kgmg.us-west-2.aws.confluent.cloud:9092]
2020-06-29T11:44:51.076-0700 [DEBUG] plugin.terraform-provider-kafka_v0.2.7: 2020/06/29 11:44:51 [DEBUG] Config @ {0xc000658660 120   ***** ***** true true XLRQDREHBRQXAQDN ***** plain}
2020-06-29T11:44:51.077-0700 [DEBUG] plugin.terraform-provider-kafka_v0.2.7: 2020/06/29 11:44:51 [INFO] Checking the diff!
2020-06-29T11:44:51.077-0700 [DEBUG] plugin.terraform-provider-kafka_v0.2.7: 2020/06/29 11:44:51 [INFO] Partitions have changed!
2020-06-29T11:44:51.077-0700 [DEBUG] plugin.terraform-provider-kafka_v0.2.7: 2020/06/29 11:44:51 Partitions is changing from 0 to 2
kafka_topic.test: Creating...
2020/06/29 11:44:51 [DEBUG] kafka_topic.test: applying the planned Create change
2020-06-29T11:44:51.079-0700 [DEBUG] plugin.terraform-provider-kafka_v0.2.7: 2020/06/29 11:44:51 [INFO] configuring bootstrap_servers {0xc000658660 120   ***** ***** true true XLRQDREHBRQXAQDN ***** plain}
2020-06-29T11:44:51.079-0700 [DEBUG] plugin.terraform-provider-kafka_v0.2.7: 2020/06/29 11:44:51 [INFO] Attempting to load from file
2020-06-29T11:44:51.079-0700 [DEBUG] plugin.terraform-provider-kafka_v0.2.7: 2020/06/29 11:44:51 [ERROR] Unable to read certificate open : no such file or directory
2020-06-29T11:44:51.079-0700 [DEBUG] plugin.terraform-provider-kafka_v0.2.7: 2020/06/29 11:44:51 [ERROR] Error creating kafka client open : no such file or directory
2020-06-29T11:44:51.079-0700 [DEBUG] plugin.terraform-provider-kafka_v0.2.7: 2020/06/29 11:44:51 [DEBUG] lazy client init open : no such file or directory; config {0xc000658660 120   ***** ***** true true XLRQDREHBRQXAQDN ***** plain}
2020/06/29 11:44:51 [DEBUG] kafka_topic.test: apply errored, but we're indicating that via the Error pointer rather than returning it: open : no such file or directory
2020/06/29 11:44:51 [ERROR] <root>: eval: *terraform.EvalApplyPost, err: open : no such file or directory
2020/06/29 11:44:51 [ERROR] <root>: eval: *terraform.EvalSequence, err: open : no such file or directory

Error: open : no such file or directory

  on pksd.tf line 12, in resource "kafka_topic" "test":
  12: resource "kafka_topic" "test" {


2020-06-29T11:44:51.094-0700 [DEBUG] plugin: plugin process exited: path=/Users/rojo/Workspace/parkside-securities/terraform-provider-kafka/examples/pksd/terraform-provider-kafka_v0.2.7 pid=21360
2020-06-29T11:44:51.094-0700 [DEBUG] plugin: plugin exited
```

**Comment by Mongey** [comment]
This should be resolved in the `0.2.8` / `0.2.9` release.

**Comment by karimSat** [comment]
thanks Connor, it works great for us !
